go.mod

```diff
@@ -1,75 +1,72 @@
-module volcano.sh/apis
+module github.com/mahdikhashan/apis
```

Comment:
@mahdikhashan revert it later when we want to merge it.

Reply:
There is no need to replace the module name here, because you forked the apis repository, you can still replace it with `github.com/mahdikhashan/apis [commit id]` in the go.mod of the Volcano main repo

Reply:
Yes we want to merge it asap

Reply:
yes, you are right. so shall i revert it when we want to merge it?

Reply:
Yes we want to merge the APIs first, please don't change the module name now